featuretools_tsfresh_primitives/utils.py

```diff
@@ -56,8 +58,9 @@ def add_primitive_instances(primitive, primitives):
             primitives.append(instance)
 
     for key in parameters:
-        if key in PRIMITIVES_SUPPORTED:
-            primitive = PRIMITIVES_SUPPORTED[key]
+        if key in PRIMITIVE_NAMES:
+            index = PRIMITIVE_NAMES.index(key)
```

Comment:
for this particular case, I think making a dictionary from `SUPPORTED_PRIMITIVES` makes sense